libs/sdk-core/src/breez_services.rs

```diff
@@ -226,17 +225,12 @@ impl BreezServices {
     ///
     /// # Arguments
```

Comment:
IMO we can get rid of the `Arguments` doc section for functions that merged all args in a single `..Request` arg.

That doc section became superfluous, the caller now goes to the `..Request` doc to read about the request args.

Reply:
Done 9d7df482d087253e02d776a4aaddc3810abc4b8f